server/chat-commands/room-settings.ts

```diff
@@ -813,6 +842,7 @@ export const commands: ChatCommands = {
 			creationTime: Date.now(),
 			modjoin: parent ? null : '+',
 			parentid: parent,
+			section: 'nonpublic',
```

Comment:
Do we need a section named `'nonpublic'`?

Reply:
no